pixi.toml

```diff
@@ -0,0 +1,34 @@
+[project]
+name = "openMVS"
+version = "0.1.0"
+description = "Add a short description here"
+channels = ["conda-forge"]
+platforms = ["linux-64"]
+
+[tasks]
+
```

Comment:
Please add task for getting igl and compile and eventually run the various binaries